firmware/lib/Axis/src/Axis.cpp

```diff
@@ -0,0 +1,264 @@
+#include "Axis.h"
+
+// -------------------------------- X AXIS SETUP -------------------------------- //
+Axis setup_axis_x()
```

Comment:
Given that all the pinMode configurations happen here, why not attach the interrupts as well?

Reply:
Yeah, i kinda move things into modules super quickly and need to give more thought to the location of different things